source/Playnite/Emulators/Definitions.yaml

```diff
@@ -7,6 +7,15 @@
       ImageExtensions: [conf]  
       ExecutableLookup: ^dosbox\.exe
 
+﻿- Name: D-Fend Reloaded (DOSBOX frontend)
+  Website: 'http://dfendreloaded.sourceforge.net/'
+  Profiles:
+    - Name: Default
+      DefaultArguments: '"{Game}"'

```

Comment:
Are you sure this will work? I would use `{Name}` variable introduced in 4.2

Reply:
Changed. Feel free to merge.